tools/setup_perf_cluster.sh

```diff
@@ -53,6 +53,10 @@ else
   fi
 fi
 
+if [[ $DEBUG == "true" ]]; then
+  sed -i '/echo-svc-deployment1/a \ \ annotations:\n\ \ \ \ sidecar.istio.io\/interceptionMode' $FNAME.yaml
```

Comment:
You need to set the annotation as
```
    annotations:
        sidecar.istio.io/interceptionMode: TPROXY
```

Reply:
how is TPROXY related to debug or not debug ?

Reply:
Its here https://github.com/istio/istio/blob/master/install/kubernetes/helm/istio/templates/sidecar-injector-configmap.yaml#L132-L136